policyengine_canada/variables/gov/provinces/ns/tax/income/income_assistance/eligibility/assets/ns_income_assistance_not_applicable_assets.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_canada.model_api import *
+
+
+class ns_income_assistance_not_applicable_assets(Variable):
+    value_type = float
+    entity = Household
+    label = "Nova Scotia income assistance not applicable assets"
+    unit = CAD
+    definition_period = YEAR
+    reference = "https://novascotia.ca/just/regulations/regs/esiaregs.htm#TOC2_4"  # Section 25
+    defined_for = ProvinceCode.NS
+
+    # None of the following is an applicable asset:
```

Comment:
@MaxGhenis do we want it to be documented this detailed?

Reply:
Detailed documentation is good, but since this is disconnected from the rest of the tree I'd suggest instead putting it into the applicable assets variable and removing this variable